config/software/datadog-agent.rb

```diff
@@ -140,8 +141,48 @@
     copy 'packaging/osx/com.datadoghq.Agent.plist.example', "#{install_dir}/etc/"
   end
 
-  # The file below is touched by software builds that don't put anything in the installation
-  # directory (libgcc right now) so that the git_cache gets updated let's remove it from the
-  # final package
-  delete "#{install_dir}/uselessfile"
+  if windows?
+    # Let's ship win32
+    copy 'win32', "#{install_dir}/agent"
+
+    # Let's ship images for our wonderful GUI too as well as an HTML template we can definitely
+    # show off with...
+    mkdir "dist"
+    copy "packaging/datadog-agent/win32/install_files/guidata", "dist"
+    copy "packaging/datadog-agent/win32/install_files/ca-certificates.crt", "#{install_dir}/agent/"
+    copy "packaging/datadog-agent/win32/install_files/license.rtf", "#{install_dir}/license.rtf"
+
+    # Let's build an exe to launch as a service (and the GUI at the same time)
+    # Note that it'd be really cool to build the service exe in Go because we wouldn't have to ship
+    # Python with it and it would save several megabytes
+    # Also if we could find a way to build GUIs (JS on Windows, Native on OSX ?) that don't require
+    # any deps, I'm pretty sure we could go under 35 Megs on Windows
+    %w(pywintypes27 pythoncom27 pythoncomloader27).each do |name|
+      copy "#{install_dir}/embedded/Lib/site-packages/pywin32_system32/#{name}.dll",
+           "#{install_dir}/embedded/Lib/site-packages/win32"
+    end
+
+    # Let's "compile" the GUI and the service
+    command "#{install_dir}/embedded/python setup.py py2exe"
+
+    copy "dist", "#{install_dir}"
+    copy "win32/status.html", "#{install_dir}/dist/status.html"
+    # Avoid shipping twice ddagent.exe
+    delete "#{install_dir}/dist/ddagent.exe"
+    # The GUI also needs to have the certificate in its folder to send e-mails via Flare
+    copy "datadog-cert.pem", "#{install_dir}/dist/datadog-cert.pem"
+
+    # Special directories, which won't be installed at the same place than others (ProgramData)
+    mkdir "../../extra_package_files"
+    mkdir "../../extra_package_files/EXAMPLECONFSLOCATION"
+
+    # This uses part of our fork of Omnibus. We copy "extra_package_files" that we want here
+    # so that they can be harvested by heat, and shipped in the MSI by light
+    copy "conf.d/*", "../../extra_package_files/EXAMPLECONFSLOCATION"
+
+    # Weight-loss surgery
+    command "#{install_dir}/embedded/Scripts/pip.exe uninstall -y PySide"
+    command "CHDIR #{install_dir} & del /Q /S *.pyc"
```

Comment:
question: Do you know what happens to the `pyc` files when we uninstall/upgrade the Agent? Just want to make sure the pyc files are handled correctly so that we never end up running `pyc` files that are actually older than their `py` files counterparts